api4/channel_test.go

```diff
@@ -915,8 +915,13 @@ func TestDeleteChannel(t *testing.T) {
 
 	_, resp = th.SystemAdminClient.DeleteChannel(publicChannel5.Id)
 	CheckNoError(t, resp)
+}
 
-	th.InitBasic().InitSystemAdmin()
+func TestDeleteChannel2(t *testing.T) {
```

Comment:
Broke this test up to avoid having to re-initialize the system admin, and more cleanly isolate the tests.